lib/backend/k8s/resources/profile.go

```diff
@@ -240,7 +246,12 @@ func (c *profileClient) Watch(ctx context.Context, list model.ListInterface, rev
 	if len(rlo.Name) != 0 {
 		log.WithField("name", rlo.Name).Debug("Watching a single profile")
 		var err error
-		if strings.HasPrefix(rlo.Name, conversion.NamespaceProfileNamePrefix) {
```

Comment:
This handles validation, but I don't see anywhere in this PR that actually sends the Profile object over the watch.

I think we need adjustments to both List() and Watch() so that they return this new allow Profile() object when called.

Reply:
We resolved this over Slack with Shaun. We'll always return the allow-all profile but never in watches.